packages/scripts/utils/config.js

```diff
@@ -268,10 +271,8 @@ function getWebpackEntryPoints() {
 		absolute: true,
 	} );
 	if ( ! entryFile ) {
-		log(
-			chalk.bold.red( 'No entry files discovered in the "src" folder.' )
-		);
-		exit( 1 );
+		log( chalk.yellow( 'No entry file discovered in the "src" folder.' ) );
```

Comment:
```suggestion
		log( chalk.yellow( 'No entry file discovered in the "src" directory.' ) );
```

`directory` would match the other warnings.